model-api/src/commonMain/kotlin/org/modelix/model/api/INodeReferenceSerializer.kt

```diff
@@ -84,6 +84,7 @@ interface INodeReferenceSerializer {
             }
         }
 
+        @Deprecated("use INodeReference.serialize()", ReplaceWith("ref.serialize()"))
```

Comment:
Any chance to lift out adding these deprecations and maybe the related refactorings into a separate commit with a separate rationale? That makes it easier to understand why they were added later on.